legal-api/gunicorn_server.py

```diff
@@ -0,0 +1,10 @@
+
+import time
+
+
+def pre_fork(server, worker):
+    # Delay loading of each worker by 5 seconds
+    # This is done to work around an issue where the Traction API is returning an invalid token. The issue happens
+    # when successive token retrieval calls are made with less than 2-3 seconds between the calls.
+    time.sleep(5)
```

Comment:
Does this delay by 5 seconds per worker?

Reply:
yes

Reply:
![image](https://github.com/bcgov/lear/assets/6685223/607e5e46-a4fd-40c3-a612-e38664de8daa)